qm-plugins/qm-vip/qm-vip.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Plugin Name: Query Monitor VIP
+ * Description: Additional collector for Query Monitor for VIP-specific information.
+ * Version: 1.0
+ * Author: Automattic, Rebecca Hum
+ */
+
+if ( ! defined( 'VIP_GO_APP_ENVIRONMENT' ) || 'local' === constant( 'VIP_GO_APP_ENVIRONMENT' ) ) {
```

Comment:
In what case do we set `'local' === constant( 'VIP_GO_APP_ENVIRONMENT' )`?

Reply:
dev-env

Reply:
I think it would be valuable to have it on because: 
a) We can bake in the version file in our mu-plugins image for dev env - the plan is to update it weekly with the release, so it may be valuable there
b) the file missing may indicate that this is a local/unbuilt copy of dev env